src/Rule/Consignment/ShipmentOptionsRule.php

```diff
@@ -9,24 +9,83 @@
 
 class ShipmentOptionsRule extends Rule
 {
+    private $requiredOptions;
+
+    private $conditionalOptions;
+
+    /**
+     * @param $required
+     * @param $conditional
```

Comment:
Hi ik zou de term 'conditional'  vervangen door iets wat meer meteen zegt wat het doet, misschien 'countrySpecific' of iets beters als je nog iets beters weet?